test/junit/scala/StringContextTest.scala

```diff
@@ -224,11 +224,7 @@ class StringContextTest {
 
       // literals and arg indexes
       f"%%" -> "%",
-      f" mind%n------%nmatter%n" ->
-       """| mind
-          |------
-          |matter
-          |""".stripMargin,
+      f" mind%n------%nmatter%n" -> String.format(" mind%n------%nmatter%n"),
```

Comment:
You can preserve the visual pun by
```
      f" mind%n------%nmatter%n".lines.mkString("\n") ->
       """| mind
          |------
          |matter
          |""".stripMargin.lines.mkString("\n"),

```
`lines.mkString` is an easy EOL normalizer.